src/api/window/window.ts

```diff
@@ -506,6 +506,16 @@ export class _Window extends EmitterBase<WindowEvents> {
                 } else {
                     reject(pageResolve);
                 }
+                try {
+                    // this is to enforce a 5.0 contract that the child's main function
+                    // will not fire before the parent's success callback on creation.
+                    // if the child window is not accessible (CORS) this contract does
+                    // not hold.
+                    const webWindow: any = this.getWebWindow();
+                    webWindow.fin.__internal_.openerSuccessCBCalled();
+                } catch (e) {
+                    console.error(e);
```

Comment:
lets no log the error as this will fail in the expected cast most times

Reply:
yeah, i tried to remove it. But the linting does't allow empty body. Can i just comment it out?

Reply:
You can tslint ignore https://palantir.github.io/tslint/rules/no-empty/

Reply:
I can change it. But i remember we use same tslint rules for core and js-adapter. Should we make them same?